### PR TITLE
Skin cancer detection

### PR DESCRIPTION
**Pull request by SoniGarima**
Sir, 
The PR is for issue #51 
The data-set I have used for training and testing the model is skin-cancer-mnist-ham10000. This data-set contains 10015 unique images of 7 types of skin cancer. The link to the data-set is [here](https://www.kaggle.com/kmader/skin-cancer-mnist-ham10000).
Here's the link to [colab notebook](https://colab.research.google.com/drive/1kyYvCBJIt5AAn_u9Oy1-LZ21kkUqEK94?usp=sharing).
I have used the mobilenet model with some trainable layers and some already trained on 'imagenet' database. Apart, I have trained it using 2 different ways:
1. Simply on whole dataset with 10 epochs --> gave accuracy of about 75% on validation set.
2. Used data augmentation ( using ImageDataGenerator API provided by Keras) on dataset with 40 epochs of training --> gave 80% accuracy on validation set.

I have used fit_generator method of Keras API to train the model and for generating the dataset from image folders, I chose flow_from_directory method of ImageDataGenerator with many parameters for data augmentation. I have split the data into 90% train data and 10% test data using train_test_split method of sklearn module.
I found second model more convincing because it is trained on more number of images and more number of times. Besides, The evaluation metrics of second model is as follows:
accuracy: 0.8042 
Precision = 0.81
Recall = 0.80
F1-Score = 0.804

### CODE REVIEW

**Comment by smaranjitghose** [comment]
Nice work @SoniGarima!
Would like to continue to integrate it with our stream app now?

**Comment by SoniGarima** [comment]
> Would like to continue to integrate it with our stream app now?

Sure!
